types/openseadragon/index.d.ts

```diff
@@ -603,30 +649,52 @@ declare namespace OpenSeadragon {
 
         blurHandler: (event: OSDEvent<any>) => void;
         clickHandler: (event: OSDEvent<any>) => void;
+        contextMenuHandler: (event: OSDEvent<any>) => void;
         dblClickHandler: (event: OSDEvent<any>) => void;
         destroy(): void;
         dragEndHandler: (event: OSDEvent<any>) => void;
         dragHandler: (event: OSDEvent<any>) => void;
         enterHandler: (event: OSDEvent<any>) => void;
         exitHandler: (event: OSDEvent<any>) => void;
+        leaveHandler: (event: OSDEvent<any>) => void;
         focusHandler: (event: OSDEvent<any>) => void;
         getActivePointerCount(): number;
         getActivePointersListByType(type: string): GesturePointList;
-        getActivePointersListsExceptType(type: string): GesturePointList[];
         keyDownHandler: (event: OSDEvent<any>) => void;
         keyHandler: (event: OSDEvent<any>) => void;
         keyUpHandler: (event: OSDEvent<any>) => void;
         moveHandler: (event: OSDEvent<any>) => void;
         nonPrimaryPressHandler: (event: OSDEvent<any>) => void;
         nonPrimaryReleaseHandler: (event: OSDEvent<any>) => void;
+        overHandler: (event: OSDEvent<any>) => void;
+        outHandler: (event: OSDEvent<any>) => void;
         pinchHandler: (event: OSDEvent<any>) => void;
         pressHandler: (event: OSDEvent<any>) => void;
+        preProcessEventHandler: (event: OSDEvent<any>) => void;
```

Comment:
preProcessEventHandler will always get a EventProcessInfo object

Reply:
> preProcessEventHandler will always get a EventProcessInfo object

Thank you @msalsbery, I think it's fixed now